deploy/docker/fs/opt/appsmith/pg-utils.sh

```
@@ -0,0 +1,92 @@
+      : 'http://' + connectionString; //Prepend a fake scheme for URL parsing
+    const url = require('url');
+    const parsedUrl = new url.URL(pgUri);
+
+    // Extract the pathname and remove the leading '/'
+    const db = parsedUrl.pathname.substring(1);
+
+    // Default the port to 5432 if it's empty
+    const port = parsedUrl.port || '5432';
+
+    console.log(\`\${parsedUrl.username || '-'} \${parsedUrl.password || '-'} \${parsedUrl.hostname} \${port} \${db}\`);
+  " "$conn_string")
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)